### ISSUE TITLE
Not using my config file

### ISSUE THREAD

**Issue by cutterbl** [CryogenicPlanet/depp#9]
I have an extensive list of externals, so I generated a `config.json` so I wouldn't have to set the config with every command.

```json
{
  "js":true,
  "path":"",
  "log":false,
  "report":true,
  "show-versions":true,
  "dev":true,
  "externals":[
    "prop-types",
    "invariant",
    "lodash/sortBy",
    "lodash/findIndex",
    "date-arithmetic",
    "memoize-one",
    "react",
    "dom-helpers/listen",
    "dom-helpers/contains",
    "dom-helpers/closest",
    "rollup-plugin-terser",
    "rollup-plugin-size-snapshot",
    "rollup-plugin-replace",
    "rollup-plugin-node-resolve",
    "rollup-plugin-commonjs",
    "webpack-atoms",
    "webpack-dev-server",
    "webpack",
    "react-tackle-box/Slot",
    "@storybook/theming",
    "@storybook/react",
    "@storybook/addon-actions/register",
    "moment"
    ],
    "ignore-namespaces":null,
    "browser-platform":false
  }
```

But when I run `depp -v -j`, my output continues to show that it 'Could not resolve "x" (mark it as external to exclude it from the bundle)'. I also get a lot of output like '{ Unexpected "<" 0xc001c1e9c0 [] <nil>}'. Am I missing something?

**Comment by CryogenicPlanet** [comment]
What is `x` here? I think the output error might be causes by the "ignore-namespaces" : null, that might not be valid json, I can test and check.

That said it would be great if you could provide a minimum reproduction example, so I can just test against it

**Comment by cutterbl** [comment]
The `x` was every external I had defined in the config. I was attempting to test depp against a clone of `react-big-calendar`. That `ignore-namespaces` line came from running the `init`.

**Comment by CryogenicPlanet** [comment]
Okay I am not running into the exact same problem but definitely noticing some problems with reading the config file, will work on a patch

**Comment by CryogenicPlanet** [comment]
Hey, the bug will be resolved when #10 is merged and you can use `v0.0.10` to test it. If you are curious what the bug(s) itself were you can check the PR.

However, wanted to point out a few things

In `react-big-calendar` this line is wrong https://github.com/jquense/react-big-calendar/blob/ad8defa643692911fd0b00c71b70de94715140a9/stories/helpers/index.js#L27 and will cause a build error.
It should be this

```tsx
export {default as resourcesEvents}  from './resourceEvents'
```

Also `externals` are packages you want `esbuild` to ignore in the build process, so if you mark a package as `external` but it used in the codebase it will show as unused. The only actual `external` package need for this codebase is `emitter` (and `react-big-calendar` itself if you building examples too)
